### PR TITLE
style(secondary button): focus and hover styling, pipe styling

### LINKED ISSUE #580: Secondary button - Styling errrors
There are styling errors with the secondary button. 

### Secondary button colors (by state)

**Default:** 
- Pacific (#0072ce)
- Includes outline, pipe character, icon, text

**Hover:** 
- Dark Pacific (#0050b4)
- Includes outline, pipe character, icon, text

**Focus:** 
- Pacific (#0072ce)
- Includes dotted outline, outline, pipe character, icon, text

**Hover/Focus:** 
- Dark Pacific (#0050b4)
- Includes dotted outline, outline, pipe character, icon, text

**Active:** 
- Navy (#254b87)
- Includes outline, pipe character, icon, text

- On focus should get a Dark Pacific (#0050b4) dotted outline
- Pipe character should be Pacific blue

## Current behavior
- Focus outline is Black
- Pipe character is Gray
Screenshot 2024-05-24 at 8 12 55 AM

## Expected behavior
- Focus outline is Pacific blue
- Pipe character is Pacific blue
Screenshot 2024-05-24 at 8 13 36 AM

### PR DESCRIPTION
**Pull request by shindigira**
closes #580 

## Changes
- style(secondary button): updated state and pipe styling
- Upload - missed small text update
- Summary - links open a new window

| Normal | Hover | Focus | Active |
|--|--|--|--|
|Screenshot 2024-06-03 at 5 56 40 PM|Screenshot 2024-06-03 at 5 53 36 PM|Screenshot 2024-06-03 at 5 53 48 PM|Screenshot 2024-06-03 at 6 05 58 PM|

## Reference
- https://github.com/cfpb/design-system-react/issues/357#issue-2316248266

### CODE REVIEW

**Review by natalia-fitzgerald — CHANGES_REQUESTED** [review]
@shindigira 
I took a look at the secondary button implementation. Can you remove the style called "Focus/Hover" and use what you show for "Focus/Hover" and use it for "Focus" instead? Also, can you show me what 2px for the solid line in the Hover and Focus states would look like? This is more consistent with what we do for inputs, and may also provide more visual contrast when a user interacts with the button. The pipe update looks good.

**Comment by natalia-fitzgerald** [comment]
@shindigira 
Ok, I updated to exclude Hover/Focus and just include Focus. https://github.com/cfpb/sbl-frontend/issues/580#issue-2315367994

**Comment by natalia-fitzgerald** [comment]
@shindigira 
Default and Active look good. Still need to nail down Hover and Focus.

File: tailwind.config.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ const config = {
         black: '#101820',
         grayDark: '#43484E',
         pacific: '#0072CE', // TODO: Integrate DS color vars
+        darkPacific: '#0050b4',
```
Comment (meissadia) [inline tailwind.config.js:21]: Easier to see related colors if this was `pacificDark`?

**Review by meissadia — DISMISSED** [review]

**Review by meissadia — APPROVED** [review]

**Pull request merged**